client/components/token-field/token-input.jsx

```diff
@@ -17,21 +18,24 @@ var TokenInput = React.createClass( {
 			onChange: function() {},
 			onBlur: function() {},
 			value: '',
-			disabled: false
+			disabled: false,
+			placeholder: '',
 		};
 	},
 
 	mixins: [ PureRenderMixin ],
 
 	render: function() {
 		const props = { ...this.props, onChange: this._onChange };
+		const { value, placeholder } = props;
+		const size = ( placeholder && placeholder.length || value.length ) + 1;
```

Comment:
Shoudn't the value length take precedence here? For example, when the input is empty and I enter a single character, I'd expect the size to revert to `2` because the placeholder is no longer shown, but it stays at `17` in the DevDocs example. That said, I don't think there's any negative effects with this current behavior.